modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java

```diff
@@ -840,72 +839,52 @@ public void testSetRoleUsers() throws Exception {
 		Assert.assertTrue(ArrayUtil.contains(user.getRoleIds(), roleId));
 	}
 
+	@FeatureFlags(enable = false, value = "LPD-36010")
 	@Test
 	public void testSortUsersByLastLoginDate() throws Exception {
-		String featureFlag = "feature.flag.LPD-36010";
+		Calendar calendar = Calendar.getInstance();
+		String middleName = "middleName";
```

Comment:
When creating a variable in which its value does not matter for the test case, use a random value. The idea is to only hard code values that matter for the test case. That makes easier to understand the test.

This isn't related to feature flag and you can ignore my comment if you want.